tests/smb/smb_deployment_declarative_method.py

```diff
@@ -84,6 +94,17 @@ def run(ceph_cluster, **kw):
             file_mount,
```

Comment:
Did we miss adding a clustering parameter for declarative deployment?

Reply:
```suggestion
            file_mount,
```

Reply:
Hi @adityaramteke 

No,

Clustering is by default enabled.

Ref: https://docs.ceph.com/en/latest/mgr/smb/

Thanks,
Mohit Bisht